test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.AzureImageCodeGeneratorTest.approved.txt

```diff
@@ -0,0 +1,159 @@
+﻿// This file was auto-generated by ML.NET Model Builder. 
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using CodeGenTest.Model;
+
+namespace CodeGenTest.ConsoleApp
+{
+    public static class ModelBuilder
+    {
+        private static string TRAIN_DATA_FILEPATH = @"C:\Users\xiaoyuz\Desktop\flower_photos_tiny_set_for_unit_tests\data.tsv";
```

Comment:
Why reference local files?

Reply:
Oh my bad, this file is outdated and shouldn't be included in repo

## Resolved